build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
         versionCode         = 48
 
         xikoloApiVersion    = 3
-        realmSchemaVersion  = 8
+        realmSchemaVersion  = 7
```

Comment:
I checked again and `realmSchemaVersion` should actually be `8` like on `master`.